docs/pages/setup/admin/github-sso.mdx

```diff
@@ -63,18 +68,66 @@ spec:
         - access
 ```
 
-The values of `client_id`, `client_secret`, and `redirect_url` come from the
-GitHub OAuth App you created earlier.
+The values of `client_id` and `client_secret` come from the GitHub OAuth App you created earlier. `redirect_url` was automatically configured based on the settings of your Teleport cluster when you ran [`tctl sso configure github`](../reference/cli.mdx#tctl-sso-configure-github).
+
+<Admonition
+  type="tip"
+  title="Teams to logins mapping"
+>
+  The `--teams-to-logins` flag (`-r` for short) can be repeated multiple times. Each repetition will create an additional entry in the `teams_to_logins` mapping of your GitHub authentication connector.
+  
+  To map a GitHub team to multiple Teleport roles, list these roles after the name of the GitHub team, separated by commas: `-r org,team,role1,role2,role3,...`.
+</Admonition>
 
-Teleport will request only the `read:org` OAuth scope. Read more about OAuth scopes in GitHub's documentation:
+Teleport will request only the `read:org` OAuth scope. Read more about OAuth scopes in [GitHub's documentation](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/).
 
 [GitHub OAuth scopes](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/)
 
-Finally, create the connector using `tctl`:
+To ensure the connector is working as intended: SSO flow is passing, roles are assigned as intended etc., you can now test it using [`tctl sso test`](../reference/cli.mdx#tctl-sso-test) command.
 
 ```code
-$ tctl create github.yaml
-# authentication connector "github" has been created
+$ tctl sso test github.yaml
+# If browser window does not open automatically, open it by clicking on the link:
+#  http://127.0.0.1:62482/e1fadd3f-dc6a-420b-9791-94fd29dfc2db
+# Success! Logged in as: ExampleGithubUser
+# --------------------------------------------------------------------------------
+# Authentication details:
+#    logins:
+#    - access
+#    roles:
+#    - access
+#    traits:
+#      github_teams:
+#      - admins
+#      kubernetes_groups: null
+#      kubernetes_users: null
+#      logins:
+#      - ExampleGithubUser
+#    username: ExampleGithubUser
+# --------------------------------------------------------------------------------
+# [GitHub] Received claims:
+# organization_to_teams:
+#   octocats:
+#   - admins
+# teams:
+# - admins
+# username: ExampleGithubUser
+# 
+# --------------------------------------------------------------------------------
+# [GitHub] Connector team to logins mapping:
+# - logins:
+#   - access
+#   organization: octocats
+#   team: admins
+# 
+# --------------------------------------------------------------------------------
+# For more details repeat the command with --debug flag.
+```
+
+The test command can accept the auth connector from standard input. The pipeline may also utilise `tee` to save the connector to file:
```

Comment:
```suggestion
The test command can accept the auth connector from standard input. The pipeline may also utilize `tee` to save the connector to a file so you can define and test the connector in a single command:
```
I think the docs mostly use US spelling, so we can standardize on that. Also linked this sentence to the preceding text a bit more.